diff --git a/android/build.gradle b/android/build.gradle
index 5ed5f9be385808ea93e6f9fbc474115089d5cf89..7169544cb5f6be5655d18d76f9cf6ae09c259418 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
   // Buildscript is evaluated before everything else so we can't use getExtOrDefault
   def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['Readium_kotlinVersion']
-  ext.readium_version = '3.1.0'
+  ext.readium_version = '3.1.2'
 
   repositories {
     google()
